prometheus/testutil/promlint/promlint.go

```diff
@@ -313,9 +313,10 @@ var (
 		// Base units.
 		"amperes": "amperes",
 		"bytes":   "bytes",
-		"celsius": "celsius", // Celsius is more common in practice than Kelvin.
+		"celsius": "celsius", // Celsius is preferred in computing.
 		"grams":   "grams",
 		"joules":  "joules",
+		"kelvin":  "kelvin", // Kelvin is used for color temperatures, not Celsius.
```

Comment:
How about: "SI base units, used in special cases (e.g. color temperature, scientific measurements)."